src/display/editor/tools.js

```diff
@@ -279,6 +280,63 @@ class ClipboardManager {
   }
 }
 
+class ColorManager {
+  static _colorsMapping = new Map([
+    ["CanvasText", [0, 0, 0]],
+    ["Canvas", [255, 255, 255]],
+  ]);
+
+  get _colors() {
+    if (typeof document !== "undefined") {
```

Comment:
Why do we need this check, since I don't believe this code is expected to run outside of browsers?
Also, in the method just below you're accessing `window` *directly* without checking if it's available first.

Reply:
I added this check to fix `gulp unittestcli` which was unhappy with that (likely because a static property is using it)... so I chose the easiest solution.

Reply:
Ah, that explains things. In that case, since this doesn't matter in browser, please see the suggestion below (which based on what we do in the viewer-code in a couple of spots).